doc-content/enterprise-only/project-data/preview-editor-contextual-menu-ref.adoc

```diff
@@ -0,0 +1,70 @@
+[id='preview-editor-contextual-menu-ref']
+= Contextual menu options
+
+The *Test Scenario (Preview)* editor provides contextual menu options which will enable you to perform basic operations on the table such as adding, deleting, and, duplicating rows and columns. To use the contextual menus, you need to right-click a table element. Menu options differ based on the table element you select.
```

Comment:
Change: 
"The *Test Scenario (Preview)* editor provides contextual menu options which will enable ..."
to 
"The *Test Scenario (Preview)* editor provides contextual menu options, which enables you to "
OR "The *Test Scenario (Preview)* editor provides contextual menu options that enables you to "

Again, not sure why is *Test Scenario (Preview)* in bold?

Reply:
Test Scenario (Preview) is a UI element hence bold.

Changed as suggested.